docs/proposals/YYYYMMDD-template.md

```diff
@@ -0,0 +1,238 @@
+---
+title: proposal Template
+authors:
+  - "@janedoe"
```

Comment:
This should probably also include reviewers and approvers

Reply:
Reason I removed it was because the assumption is that reviewers can be anyone that collaborates on the project and approvers would follow the same rules for PR approvers.

Reply:
Added!